apps/src/p5lab/P5Lab.js

```diff
@@ -519,6 +505,66 @@ P5Lab.prototype.init = function(config) {
   return loader;
 };
 
+/**
+ * Load any necessary missing animations. For now, this is mainly for
+ * the set background to block, which needs there to be backgrounds in the
+ * animation list at the start in order to look not broken,
+ * @param {Boolean} useConfig
+ * @param {Object} initialAnimationList
+ */
+P5Lab.prototype.loadMissingAnimationDefaultsIfNecessary = function(
+  useConfig,
+  initialAnimationList
+) {
+  if (this.isSpritelab) {
+    let configDictionary = {};
+    initialAnimationList.orderedKeys.forEach(key => {
+      const name = initialAnimationList.propsByKey[key].name;
+      configDictionary[name] = key;
+    });
+    if (useConfig && this.startAnimations) {
+      // We need to make sure we include all of the default animations (in case they have changed since this project was created).
+      this.startAnimations.orderedKeys.forEach(key => {
+        const name = this.startAnimations.propsByKey[key].name;
+        if (!configDictionary[name]) {
+          initialAnimationList.orderedKeys.push(key);
+          initialAnimationList.propsByKey[
+            key
+          ] = this.startAnimations.propsByKey[key];
+        }
+      });
+    }
+    // Check if initialAnimationList has backgrounds. If it doesn't add it from defaultSprites.json
```

Comment:
2 small nits here around "If it doesn't add it" 

1. what does the latter "it" mean here?
2. add a comma after "doesn't" and periods at the end of comments
```suggestion
    // Check if initialAnimationList has backgrounds. If it doesn't, add it from defaultSprites.json.
```